apps/language_server/test/source_file/experimental_text.exs

```diff
@@ -0,0 +1,778 @@
+defmodule ElixirLS.LanguageServer.SourceFile.ExperimentalTest do
```

Comment:
Is it an accidental commit?

Reply:
yep sorry about that, this branch got rebased onto the wrong branch.